Kbs.Wpf/BoatType/Update/UpdateBoatTypeViewModel.cs

```diff
@@ -0,0 +1,84 @@
+using System.Collections.ObjectModel;
+using Kbs.Business.BoatType;
+using Kbs.Wpf.BoatType.CreateBoatType;
+using Kbs.Wpf.Components;
+
+namespace Kbs.Wpf.BoatType.Update;
+
+public class UpdateBoatTypeViewModel : ViewModel
+{
+    private string _name;
+    private BoatTypeRequiredExperience _requiredExperience;
+    private BoatTypeSeats _seats;
+    private int _speed;
+    private bool _hasSteeringWheel;
+    private string _experienceErrorMessage;
+    private string _speedErrorMessage;
+    private string _seatsErrorMessage;
+    private CreateBoatExperienceViewModel _selectedExperience;
+    private CreateBoatSeatsViewModel _selectedSeats;
+    
+    
+    public ObservableCollection<CreateBoatExperienceViewModel> PossibleExperiences { get; } = [];
+    public ObservableCollection<CreateBoatSeatsViewModel> PossibleSeats { get; } = [];
```

Comment:
Eigen viewmodels maken voor de update, je herbruikt hier de Create- models.
Wat ook kan is deze ViewModels naar een Components-folder verplaatsen en ze hernoemen naar actie-loze ViewModels. (`BoatTypeExperienceViewModel` en `BoatTypeSeatsViewModel`)